lib/utils.js

```diff
@@ -91,31 +134,31 @@ function Utils(config) {
             throw new Error('Unlocated features can not be stored.');
         }
 
-        var info = Metadata(config.dyno, dataset).getFeatureInfo(f);
-        var buf = geobuf.featureToGeobuf(f).toBuffer();
-        var tile = tilebelt.bboxToTile([info.west, info.south, info.east, info.north]);
-        var cell = tilebelt.tileToQuadkey(tile);
+        var bounds = extent(f);
+        var info = {
+            bounds: bounds,
+            west: bounds[0],
+            south: bounds[1],
+            east: bounds[2],
+            north: bounds[3]
+        };
+        var buf = utils.encodeFeature(f);
         var useS3 = buf.length >= config.MAX_GEOMETRY_SIZE;
```

Comment:
Given that we don't utilize the S3 side of this as was envisioned originally, and given that this is such a breaking PR anyway, we should consider removing this stuff entirely.